amberly-awang617/week-3/giffaw-master/scripts/app.js

```diff
@@ -0,0 +1,40 @@
+// [x] make an Ajax call fire when your page loads.
+// [x] build functions that generate HTML Strings and append() them to the page
+
+// Setting up the api call variables
+var gifUrl = 'http://api.giphy.com/v1/gifs/search?q=';
+
+var key = 'e0WvxxQTfbDbQ2wiBJqywrFbZ2L8Ccek';
```

Comment:
I like that you set up your key in your JS file for giffy, but.. there was a key in your html file already. So you are receiving an error from sending a dual request to giffy api. :) Always good to read through the code provided to make sure you don't give yourself extra work. In a professional environment, this will happen a lot.